latex/contest-problem.tex

```diff
@@ -4,6 +4,7 @@
 	\renewcommand{\problemauthor}{{%problemauthor%}}
 	\renewcommand{\timelimit}{{%timelimit%}}
 	\input{{%builddir%}/samples.tex}
+	\input{{%builddir%}/constants.tex}
```

Comment:
Can/should we scope the constants, so that they don't acidentally leak into the next problem?

Reply:
`\begingroup` and `\endgroup` should already scope this